changelog.json

```
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "2.3.0": [
+      "[New] Prompt users to make a fork if they fail to push to a GitHub repository due to permissions  - #8548",
+      "[Added] Show link to upstream pull requests in pull request list - #8965",
+      "[Added] Show a commit warning if the user can't push to the repository - #8938",
+      "[Added] Users can configure Desktop to always bring or leave changes when switching branches - #7746",
+      "[Added] Prompt users to re-authorize if required by any of their GitHub organizations - #8910",
+      "[Added] Support GoLand as external editor on Mac - #8802. Thanks @thinklinux!",
+      "[Fixed] Remove stray characters from Git progress indicators - #8962",
+      "[Fixed] Show proper dialog when push fails due to missing workflow scope - #8958",
+      "[Fixed] Route the user to the correct URL when opening a pull request from a fork in the browser - #8928",
```

you can take or leave this one, i'm on the fence

```suggestion
      "[Fixed] Route the user to the forked pull request URL when opening a pull request from a fork within Desktop - #8928",
```